profile_specs/profiles.json

```diff
@@ -21,6 +21,7 @@
         "2016-09-01": ["*"]
       },
       "Microsoft.Policy" : {
```

Comment:
should this be pointing to the specific Policy profile, defined in the policy gem?
similarly for the rest for the rest of the entries in the file?
Is Azure Stack profile definition done at package level? spec level? (I'm asking about the source from where we are getting the info)